simulation/README.md

```diff
@@ -44,21 +35,16 @@ Make sure you have some version of python installed on your personal computer. *
 
 ## Running the Simulation
 ### Docker Simulation Setup
-1. Open up a terminal and ssh into your Docker Container with the `-XY` flag (i.e. run `ssh -XY -p 2200 duke@[ip address]`).
-2. Run `source /opt/ros/kinetic/setup.bash` and `roscore &`.
+1. Open up a terminal and ssh into your Docker Container with the `-XY` flag (i.e. run `ssh -XY -p 2201 root@localhost`).
+2. Run `source /opt/ros/melodic/setup.bash` and `roscore &`.
 3. Run `cd dev/robosub-ros/simulation` and then `./runSim.sh &` (you may first need to run `chmod +x runSim.sh`). Wait until the terminal says `Initialization successful.` If it delays on the video compression library or meshcalc for an extended period of time, press enter a couple of times. This may be nothing more than confirmation-bias superstition.
 4. Run whatever ROS topic publishing code you have. In `robosub-ros/simulation/docker`, there is a python script `squareCommand.py` that you can run to make the robot move approximately in a square. (In reality, the robot will spin in wide circles because the robot isn't balanced.) You can use this script to test if communication between the simulations is working.
 
 ### Personal Computer Simulation Setup
 1. Open CoppeliaSim.
 2. Go to `File>Open Scene...` and open `empty_scene.ttt` in `robosub-ros/simulation/personal/scenes`.
 3. Press the play button to start the simulation. The robot should start bobbing up and down.
-4. On your computer, run `ros_coppelia_comm.py` at `robosub-ros/simulation/personal`. **If you have a Mac,** you must run this file with Python 3. **If you have Docker Toolbox (i.e. you have Windows, but not Windows 10 Pro, Education, or Enterprise)**, you need to run `ros_coppelia_comm.py` with the command line argument of the IP address of the Docker container (e.g. `ros_coppelia_comm.py 192.168.99.100`). By default this is `192.168.99.100`, but you can find the IP address by running `docker-machine ip` in your Docker Toolbox terminal.
-    <details>
-        <summary>Click for Explanation</summary>        
-        The reason for this is that Docker Toolbox handles the containers, and more specifically, their IP addresses, differently, requiring a different IP address. `192.168.99.100` is the default output of `docker-machine ip`, which is why it is used. If the simulation fails to connect, run `docker-machine ip` to see if the output is the IP address above.        
-    </details>
-
+4. On your computer, run `ros_coppelia_comm.py` at `robosub-ros/simulation/personal`. **If you have a Mac,** you must run this file with Python 3.
```

Comment:
Rather than specify Python 3 like that, maybe just change the command to include Python 3. Everyone should be using Python 3 now anyway. I've reworded it here for some other clarity.
```suggestion
4. In `robosub-ros/simulation/personal` on your local machine, run `python3 run_coppelia_comm.py`.
```